ngraph/frontend/frontend_manager/include/frontend_manager/place.hpp

```diff
@@ -94,12 +94,62 @@ namespace ngraph
             /// place
             virtual std::vector<Ptr> get_consuming_operations(int outputPortIndex) const;
 
+            /// \brief Returns references to all operation nodes that consume data from this place
+            /// for specified output port
+            ///
+            /// \note It can be called for any kind of graph place searching for the first consuming
+            /// operations.
+            ///
+            /// \param outputName If a given place is itself an operation node, this specifies name
+            /// of output port group
+            ///
+            /// \return A vector with all operation node references that consumes data from this
+            /// place
+            virtual std::vector<Ptr>
```

Comment:
Why not `const std::vector<Ptr>&`?

Reply:
We can't return reference of created local object, e.g.
![image](https://user-images.githubusercontent.com/21181928/126757362-ec0f183f-8424-407d-ad65-6a4c817a7d3b.png)